documentation/src/main/asciidoc/backend-elasticsearch.asciidoc

```diff
@@ -368,10 +368,26 @@ or `yyyy\|\|yyyyyyyyy` (ES6 and below)
 |java.time.YearMonth|`date` with format
 `uuuu-MM` (ES7 and above)
 or `yyyy-MM\|\|yyyyyyyyy-MM` (ES6 and below)
-|java.time.MonthDay|`date` with format `--MM-dd`
+|java.time.MonthDay|`date` with format `uuuu-MM-dd` (ES7 and above)
+or `yyyy-MM-dd` (ES6 and below).
+**The year is always set to 0**.
 |org.hibernate.search.engine.spatial.GeoPoint|`geo_point`
 |====
 
+jshell> java.time.Instant.ofEpochMilli( Long.MIN_VALUE )
```

Comment:
I would format this as Monospaced font

Reply:
Wow, this shouldn't be in there. Good catch, I'll remove it.